sparrowpy/geometry.py

```diff
@@ -465,7 +465,7 @@ def _calculate_normals(points: np.ndarray):
 
     normals = np.empty((points.shape[0],3))
 
-    for i in numba.prange(points.shape[0]):
+    for i in prange(points.shape[0]):
```

Comment:
```suggestion
    for i in numba.prange(points.shape[0]):
```

this should be fixed on main first